sympy/core/tests/test_relational.py

```diff
@@ -459,6 +459,13 @@ def test_nan_complex_inequalities():
         assert_all_ineq_raise_TypeError(r, nan)
 
 
+def test_complex_infinity_inequalities():
+    raises(TypeError, lambda: zoo > 0)
+    raises(TypeError, lambda: zoo >= 0)
+    raises(TypeError, lambda: zoo < 0)
+    raises(TypeError, lambda: zoo <= 0)
+
+
```

Comment:
1.  suggest `assert_all_ineq_raise_TypeError`: there are lots of paths through this code and I don't trust comparison with zero to capture them all.

2.  Personally, I would do something more like `test_nan_inequality_raise_errors()` above, but @smichr might not agree.